apps/antalmanac/src/components/Calendar/CourseCalendarEvent.tsx

```diff
@@ -296,6 +297,12 @@ const CourseCalendarEvent = (props: CourseCalendarEventProps) => {
                         </Link>
                     </div>
                 )}
+                {term && (
```

Comment:
This should be styled in the same way as normal sections' pop-ups, i.e., without a colon and in two columns. 

I know that this follows the styling of the location in this popup, but that was a mistake. So, while you're at it, could you make the same change to the location row? Thanks!